main/ejml-simple/src/org/ejml/simple/SimpleMatrix.java

```diff
@@ -26,7 +26,8 @@
 
 /**
  * <p>
- * {@link SimpleMatrix} is a wrapper around {@link DMatrixRMaj} that provides an
+ * {@link SimpleMatrix} is a wrapper around a primitive matrix type (e.g. {@link DMatrixRMaj} or
+ * {@link FMatrixSparseCSC}) that provides an
```

Comment:
Replace e.g. to avoid the javadoc snippet thinking this is the end of the first sentence:

```suggestion
 * {@link SimpleMatrix} is a wrapper around a primitive matrix type
 * (for example, {@link DMatrixRMaj} or {@link FMatrixSparseCSC}) that provides an
```